templates-java/src/main/java/br/com/templates_java/config/jms_aws_sqs/SQSPropertiesGenerator.java

```diff
@@ -21,6 +21,7 @@ public File runGenerate() throws IOException {
                 "cloud.aws.region.static=us-east-1\n" +
                 "cloud.aws.stack.auto=false\n" +
                 "cloud.aws.sqs.queue-name=my-queue.fifo");
+        this.generatorOptions.setName("application.properties");
```

Comment:
Hardcoded region and queue name can be a security risk, consider using environment variables or a secure configuration mechanism.